tendrl/gluster_integration/manager/__init__.py

```diff
@@ -78,6 +80,39 @@ def shutdown(signum, frame):
             {"message": "Signal handler: stopping"}
         )
         complete.set()
+        # Remove the node's name from gluster server tag
+        try:
+            gl_srvr_list = NS._int.client.read(
+                "/indexes/tags/gluster/server"
+            ).value
+            gl_srvr_list = json.loads(gl_srvr_list)
+            if NS.node_context.node_id in gl_srvr_list:
+                gl_srvr_list.remove(NS.node_context.node_id)
+            NS._int.client.write(
+                "/indexes/tags/gluster/server",
+                gl_srvr_list
```

Comment:
youll need to convert "gl_srvr_list" back to json

Reply:
its a python list only at this moment

Reply:
You need to convert "gl_srvr_list" to json before writing to etcd.

https://github.com/Tendrl/node-agent/blob/master/tendrl/node_agent/node_sync/services_and_index_sync.py#L146

Reply:
ok